timesketch/api/v1/resources/searchtemplate.py

```diff
@@ -100,21 +100,19 @@ def get(self, searchtemplate_id):
         Returns:
             Search template in JSON (instance of flask.wrappers.Response)
         """
-        form = request.json
-        parameters = {}
-
-        if form:
-            parameters = form.get("parameters", {})
-
+        form = request.json or {}
         searchtemplate = SearchTemplate.query.get(searchtemplate_id)
         if not searchtemplate:
             abort(HTTP_STATUS_CODE_NOT_FOUND, "Search template was not found")
 
         try:
             template = jinja2.Template(searchtemplate.query_string)
-            parsed_query_string = template.render(parameters)
+            parsed_query_string = template.render(form)
+            parsed_query_string = parsed_query_string.replace("\\/", "\\/")
+            parsed_query_string = parsed_query_string.replace(".", "\\.")
         except jinja2.exceptions.TemplateSyntaxError as e:
             abort(HTTP_STATUS_CODE_BAD_REQUEST, f"Search template syntax error: {e}")
+
         result_dict = {"query_string": parsed_query_string}
         return jsonify({"objects": [result_dict], "meta": {}})
```

Comment:
here you could simplify to:

`return jsonify({"objects": [{"query_string": parsed_query_string}], "meta": {}})`

Reply:
I opt to keep it as i find it more readable.